### PR TITLE
www/qt5-webengine: Update to 5.12.2

### PR DESCRIPTION
**Pull request by knobix**
This is equivalent to the latest version of D18757 (except for the Makefile and distinfo)

### CODE REVIEW

File: www/qt5-webengine/Makefile [hunk 1]
```diff
@@ -73,7 +73,7 @@ SNDIO_VARS_OFF=		QMAKE_CONFIGURE_ARGS+=-no-sndio
 # We pass `norecursive' to USES=qmake because src/plugins/plugins.pro checks
 # whether webenginewidgets is available, which fails when qmake processes all
 # .pro files at once.
-USES=		gperf jpeg python:2.7,build pkgconfig \
+USES=		gl gnome gperf jpeg python:2.7,build pkgconfig \
```
Comment (rakuco) [inline www/qt5-webengine/Makefile]: Why do we need `USES=gnome`?

**Comment by rakuco** [comment]
Could you rebase this on top of master now that 5.12.1 has landed?

File: www/qt5-webengine/Makefile (hunk 1 above)
Comment (rakuco) [reply www/qt5-webengine/Makefile]: Oops, I didn't see this is already in the tree now.

**Comment by knobix** [comment]
I'm going to clean up my mess and close this PR for now. I'll open a new PR in some minutes.

**Pull request closed without merging**